### PR TITLE
Rename safety region to vr in codebase

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Rename everything from safety region to vr in the codebase to keep consistently

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/components/choropleth/hooks/use-region-municipalities.ts [hunk 1]
```diff
@@ -15,6 +15,6 @@ export function useRegionMunicipalities(
       return;
     }
 
-    return getSafetyRegionMunicipalsForMunicipalCode(municipalCode);
+    return getVrMunicipalsForMunicipalCode(municipalCode);
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/components/choropleth/hooks/use-region-municipalities.ts:18]:
```suggestion
    return getVrMunicipalsForGmCode(gmCode);
```

File: packages/app/src/pages/veiligheidsregio/[code]/index.tsx [hunk 2]
```diff
@@ -20,8 +20,8 @@ export const getStaticProps = createGetStaticProps(
   selectVrPageMetricData()
 );
 
-const SafetyRegion = (props: StaticProps<typeof getStaticProps>) => {
-  const { selectedVrData: data, lastGenerated, safetyRegionName } = props;
+const Vr = (props: StaticProps<typeof getStaticProps>) => {
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/veiligheidsregio/[code]/index.tsx:23]:
```suggestion
const VrIndexPage = (props: StaticProps<typeof getStaticProps>) => {
```

File: packages/app/src/pages/veiligheidsregio/index.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export const getStaticProps = createGetStaticProps(
   })
 );
 
-const SafetyRegion = (props: StaticProps<typeof getStaticProps>) => {
+const Vr = (props: StaticProps<typeof getStaticProps>) => {
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/pages/veiligheidsregio/index.tsx:35]:
```suggestion
const VrIndexPage = (props: StaticProps<typeof getStaticProps>) => {
```

File: packages/app/src/data/gm.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 type MunicipalityInfo = {
   name: string;
-  safetyRegion: string;
+  vrCode: string;
   gemcode: string;
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/data/gm.ts:4]: Could you try to search/replace all occurrences of "gemcode" with "gmcode"?

**Review by VWSCoronaDashboard10 — APPROVED** [review]

File: packages/app/src/components/choropleth/hooks/use-region-municipalities.ts (hunk 1 above)
Comment (VWSCoronaDashboard14) [reply packages/app/src/components/choropleth/hooks/use-region-municipalities.ts:18]: Yeah, I'll still need to do the municipality ones. Changing that would come next.

File: packages/app/src/data/gm.ts (hunk 4 above)
Comment (VWSCoronaDashboard14) [reply packages/app/src/data/gm.ts:4]: I'll do that when converting everything related to the municipalities

File: packages/app/src/components/choropleth/hooks/use-region-municipalities.ts (hunk 1 above)
Comment (VWSCoronaDashboard10) [reply packages/app/src/components/choropleth/hooks/use-region-municipalities.ts:18]: ok, makes sense!

**Pull request merged**